controller.go

```diff
@@ -106,7 +119,7 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type sandboxTable map[types.UUID]*sandbox
```

Comment:
I am fine to removing it, but If we do it for sandbox, then let's do it for networks and endpoints as well.